File: src/app/models/quiz.ts
export class Quiz {
  id?: number;
  question: string;
  option1: string;
  option2: string;
  option3: string;
  option4: string;

  constructor(
    id: number,
    question: string,
    option1: string,
    option2: string,
    option3: string,
    option4: string
  ) {
    this.id = id;
    this.question = question;
    this.option1 = option1;
    this.option2 = option2;
    this.option3 = option3;
    this.option4 = option4;
  }
}

File: src/main/java/com/example/amigos/entities/Quiz.java
package com.example.amigos.entities;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
@Table( name = "quizzes")
public class Quiz {

    @Id
    @GeneratedValue
    Long id;

    String question;

    String option1;

    String option2;

    String option3;

    String option4;
}
